dist/bm-angular-camera.js

```diff
@@ -130,12 +131,18 @@ function CameraController($scope, $element, Camera) {
     });
   };
 
+  vm.clearImage = function clearImage() {
+    if ($window.confirm(CLEAR_CONFIRM)) {
```

Comment:
Would you be better to expose an optional `beforeClear()` function via a `&?` binding? Implementations might want to use a nice bootstrap or material model for confirmation.